### PR TITLE
Create covid statistics capstone

### PR DESCRIPTION
**Pull request by Lembani**
### Documentation: Link to [Loom video](https://www.loom.com/share/d938c7bfa5814b3fbf73e64d26e5730a)

 > Highlights:
- Home page
1. When the page loads, the web app shows the list of countries by name as well as the population that comes from the API.
2. When the user clicks (or taps) on a country, the application navigates to the details page.

- Details page
1. On the details page, the web app retrieves data from the API to show detailed covid-19 data about a specific country.
2. When the user clicks on the "Back" button (<), the user navigates to the home page.

- Tesing 
- Linters 
- Readme documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# COVID Stats
+# COVID Statistics
```
Comment (danielufeli) [inline README.md:4]: - [x] As a requirement you need to deploy a live version for your project, kindly fix the correct GitHub Pages URL for your live demo link, the one there is redirecting to a 404 page.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Lembani , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

### Highlights

- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code ✔️
- Clean Design ✔️

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (Lembani) [reply README.md:4]: Thanks for the feedback @danielufeli , Let me do it right now.

Comment (Lembani) [reply README.md:4]: It has now been resolved. 😄

**Review by danielufeli — APPROVED** [review]
Hi @Lembani , 

a

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by Lembani** [comment]
Hi @danielufeli, thank you for approving my PR. 🙏🏾

**Pull request merged**